packages/ui/src/components/sign-in/sign-in.tsx

```diff
@@ -208,7 +208,7 @@ export function SignInComponentLoaded() {
                             <button
                               type='button'
                               className='text-accent-9 focus-visible:ring-default size-4 rounded-sm outline-none focus-visible:ring-2'
-                              aria-label='Edit email address'
+                              aria-label='Start again'
```

Comment:
Are there localized values for these already by chance?

Reply:
Not yet! Already created a follow-up ticket to come back to this (and other areas) before we ship!

https://linear.app/clerk/issue/DES-1118/common-ensure-all-componentscommonprimitives-are-compatible-with-i18n